region/maridia/inner-green/Oasis.json

```diff
@@ -2649,12 +3774,73 @@
         ]}
       ],
       "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
       "note": "Spark vertically through the bomb blocks. It is still possible to break the blocks while below 30 Energy by jumping before sparking.",
       "devNote": [
         "It is possible to save some energy by sparking into the left wall, but it won't work with midair sparking, which saves the same amount and is easier.",
         "Note that it is possible to spark into the ceiling to break the blocks without energy loss, but it has collision oscillation.",
         "FIXME: Add temporary blue strats."
       ]
+    },
+    {
+      "id": 18,
+      "link": [8, 9],
+      "name": "Diagonal Shinespark to the Center",
+      "startsWithShineCharge": true,
+      "requires": [
+        {"shineChargeFrames": 50},
+        {"shinespark": {"frames": 23, "excessFrames": 10}}
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "note": "Spark diagonally through the bomb blocks."
+    },
+    {
+      "link": [9, 4],
+      "name": "Base",
+      "requires": [
+        {"or": [
+          "Gravity",
+          {"and": [
+            "canSuitlessMaridia",
+            {"or": [
+              "HiJump",
+              "canSpringBallJumpMidAir"
+            ]}
+          ]}
+        ]}
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [9, 4],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 5, "excessFrames": 3}}
+      ],
+      "flashSuitChecked": true,
+      "devNote": [
+        "FIXME: It is possible to use a flash suit to get through the top door with a diagonal spark."
+      ]
+    },
+    {
+      "link": [9, 6],
+      "name": "Base",
+      "requires": [],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [9, 9],
+      "name": "Unlock Door",
+      "requires": [],
+      "unlocksDoors": [
+        {"nodeId": 3, "types": ["powerbomb"], "requires": []}
```

Comment:
Everything works from the right side.

Reply:
Oh neat, wasn't thinking about the asymmetry with left-facing shots.